rules/private/phases/phase_zinc_compile.bzl

```diff
@@ -37,6 +37,7 @@ def phase_zinc_compile(ctx, g):
     ]
 
     zincs = [dep[_ZincInfo] for dep in ctx.attr.deps if _ZincInfo in dep]
+    scalacopts = ctx.attr.scalacopts + g.semanticdb.scalacopts
```

Comment:
Nit: should we name this something other than scalacopts, so it's obvious that it's the combination of the regular scalacopts + the semanticdb opts? Perhaps `combined_scalacopts`?

Reply:
Good idea.